lib/Doctrine/ORM/Mapping/Builder/FieldBuilder.php

```diff
@@ -55,6 +55,11 @@ class FieldBuilder
     private $sequenceDef;
 
     /**
+     * @var string
+     */
+    private $customIdGenerator;
```

Comment:
Yes, null is unset, so it doesn't do anything on `build()`. Other values must be FQCN strings, as described in the setter comment.